src/Controller/Admin/Asset/AssetHelperController.php

```diff
@@ -758,6 +759,11 @@ public function doExportAction(Request $request): JsonResponse
             stream_copy_to_stream($fileStream, $temp, null, 0);
 
             $firstLine = true;
+            if ($request->get('initial') && $header === 'no_header') {
+                array_shift($csv);
```

Comment:
```suggestion
```

this is uneccessary, because it's already considered by `getCsvData` on
https://github.com/pimcore/admin-ui-classic-bundle/pull/360/files#diff-4547f82eae3e18cc6e0476109b6312b1ed287fc51ad77ba2e9f9e7a7f18c1dd4R821
and won't push it to `$csv`

current state it would export n-1 items when selecting no title